packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/contact/template.hbs

```diff
@@ -4,7 +4,7 @@
     href={{url-for 'chat.privately-with' @contact.id}}
     {{did-insert (perform this.findRelevantMessages)}}
     {{on 'click' (prevent-default this.onClick)}}
-    class={{if this.isActive 'is-active'}}
+    class={{if this.isActive 'is-active contact-display' 'contact-display'}}
```

Comment:
You could make the value of class a string and have contact-display appear before the curlies, so it only appears once